Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -0,0 +1,108 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  SUPPORTED_REPOS = (
+      %w[brew core cask] +
+      OFFICIAL_CMD_TAPS.keys.map { |t| t.delete_prefix("homebrew/") } +
+      OFFICIAL_CASK_TAPS
+    ).freeze
+
+  sig { returns(CLI::Parser) }
+  def contributions_args
+    Homebrew::CLI::Parser.new do
+      usage_banner "`contributions` [<email>]"
+      description <<~EOS
+        Contributions to Homebrew repos for a user.
+      EOS
+
+      flag "--from=",
+           description: "Date (ISO-8601 format) to start searching contributions."
+
+      flag "--to=",
+           description: "Date (ISO-8601 format) to stop searching contributions."
+
+      comma_array "--repos=",
+                  description: "The Homebrew repositories to search for contributions in. " \
+                               "Comma separated. Pass `all` to search all repos. " \
+                               "Supported repos: #{SUPPORTED_REPOS.join(", ")}."
+
+      named_args :email, number: 1
+    end
+  end
+
+  sig { returns(NilClass) }
+  def contributions
+    args = contributions_args.parse
+
+    commits = 0
+    coauthorships = 0
+
+    all_repos = args[:repos].first == "all"
```

Comment:
```suggestion
    all_repos = args.repos.first == "all"
```

and all other `args` usage